interpreter/cling/include/cling/Utils/Utils.h

```diff
@@ -0,0 +1,33 @@
+//--------------------------------------------------------------------*- C++ -*-
+// CLING - the C++ LLVM-based InterpreterG :)
+// author: Guilherme Amadio <amadio@cern.ch>
+//
+// This file is dual-licensed: you can choose to license it under the University
+// of Illinois Open Source License or the GNU Lesser General Public License. See
+// LICENSE.TXT for details.
+//------------------------------------------------------------------------------
+
+#include <cstdlib>
+
+#ifndef CLING_UTILS_UTILS_H
+#define CLING_UTILS_UTILS_H
+
+namespace cling {
+  namespace utils{
+    // Convert string str to a boolean value
+    static inline bool isTrue(const char *str) {
```

Comment:
That does not seem to handle capital letters. Maybe cut and paste from `ROOT::FoundationUtils::CanConvertEnvValueToBool`.

Reply:
I thought about using `strcasecmp`, but Windows doesn't have that. I can copy over from `ROOT::FoundationUtils::CanConvertEnvValueToBool`, no problem. Should I also rename it in cling to something else? I don't care about the name, just wanted to make a function for this as it may come in handy in other parts of the code.

Reply:
I made it case insensitive now, but I preferred to keep my implementation, as I wanted to be able to accept `nullptr` (for which I return false). Using `std::string` needs to bring in `<string>`, `<algorithm>`, and `<cctype>`, so I thought it would be easier to just adapt my current implementation.